src/main/java/games/strategy/util/Version.java

```diff
@@ -110,33 +133,17 @@ public boolean equals(final Object o) {
     return compareTo(o) == 0;
   }
 
-  public boolean equals(final Object o, final boolean ignoreMicro) {
-    return compareTo(o, ignoreMicro) == 0;
-  }
-
   @Override
   public int hashCode() {
     return this.toString().hashCode();
   }
 
   @Override
   public int compareTo(final Object o) {
-    return compareTo(o, false);
-  }
-
-  public int compareTo(final Version other) {
-    return compareTo(other, false);
-  }
-
-  private int compareTo(final Object o, final boolean ignoreMicro) {
-    if (o == null) {
+    if (o == null || !(o instanceof Version)) {
```

Comment:
Nit: `o instanceof Version` will return `false` if `o` is `null`, so the `o == null` check is redundant.